app/src/components/market/sections/market_profile/market_status/closed.tsx

```diff
@@ -348,15 +347,8 @@ const Wrapper = (props: Props) => {
     cpk?.address.toLowerCase(),
   )
 
-  const realitioAnswerNumber = Number(formatBigNumber(realitioAnswer || new BigNumber(0), STANDARD_DECIMALS))
-
-  const scalarLowNumber = Number(formatBigNumber(scalarLow || new BigNumber(0), STANDARD_DECIMALS))
-  const scalarHighNumber = Number(formatBigNumber(scalarHigh || new BigNumber(0), STANDARD_DECIMALS))
-
   const unclampedFinalAnswerPercentage =
-    realitioAnswer && realitioAnswer.eq(MaxUint256)
-      ? 0.5
-      : (realitioAnswerNumber - scalarLowNumber) / (scalarHighNumber - scalarLowNumber)
+    realitioAnswer && realitioAnswer.eq(MaxUint256) ? 0.5 : Number(balances[1].payout.toString())
```

Comment:
Ahhh yes `balances[1].payout` is exactly what I should've used in the first place. Now that we have access to a `Big` type value, probably safer if we use the `Big` type directly in `scalarComputeEarnedCollateral`. Then maybe keep a `number` type variable as well for the display side